Setting-Up-Ubuntu-For-Programming.md

```diff
@@ -0,0 +1,219 @@
+# Setting up Ubuntu for Programming
+
+> Note: We will be referring to Ubuntu 16.04 LTS (Xenial Xerus) in this article.
+
+1. Download and Install Ubuntu Desktop
```

Comment:
I think your first point here should be to install build-essential. One needs to make sure their GCC and g++ are up to date, for any other software to be installed.